src/test/java/org/seqdoop/hadoop_bam/TestBAMInputFormat.java

```diff
@@ -80,6 +80,27 @@ public void testMultipleSplits() throws Exception {
   }
 
   @Test
+  public void testMultipleSplitsBaiEnabled() throws Exception {
```

Comment:
This test doesn't test the addBAISplits codepath, it just tests that when there is no .bai file and the bai split calculator is enabled it falls back to finding probabilistic splits.

Can you add a positive test for the .bai file case?